assert/assertions.go

```diff
@@ -927,6 +927,26 @@ func containsElement(list interface{}, element interface{}) (ok, found bool) {
 		return true, false
 	}
 
+	if listKind == reflect.Slice {
+		elementValue := reflect.ValueOf(element)
+		if listType.Elem().Kind() == reflect.Uint8 {
+			if elementType, ok := element.(string); ok {
+				return true, bytes.Contains(listValue.Bytes(), []byte(elementType))
+			}
+			if elementType, ok := element.([]byte); ok {
+				return true, bytes.Contains(listValue.Bytes(), elementType)
+			}
```

Comment:
Please use a switch with type assertion here


```suggestion
			switch elementType := element.(type) {
				case string:
				return true, bytes.Contains(listValue.Bytes(), []byte(elementType))
			
				case []byte:
				return true, bytes.Contains(listValue.Bytes(), elementType)
			}
```
 
this is pseudo code written on a phone, please adapt 😬